NewForce-at-Mountwest/bangazon-workforce-bucatini
# Bf department get all

# Description

I added functionality to allow user to a list of all departments in database.

## Related Ticket(s)
5

## Steps to Test
Once you have pulled down, press ctrl+f5
You should be able to click on the navlink for departments 
Then see all departments in a list if they do or don't have employees



# Checklist:
